src/iperf_pthread.h

```diff
@@ -10,12 +10,12 @@
  */
 
 #define PTHREAD_CANCEL_ASYNCHRONOUS 0
-#define PTHREAD_CANCEL_ENABLE NULL
```

Comment:
error: incompatible pointer to integer conversion passing 'void *' to parameter of type 'int' [-Wint-conversion]